qa/workunits/rbd/rbd-nbd.sh

```diff
@@ -360,4 +360,23 @@ _sudo dd if=${DATA} of=${TEMPDIR}/mnt/test1 bs=1M count=1 oflag=direct
 _sudo umount ${TEMPDIR}/mnt
 unmap_device ${DEV} ${PID}
 
+# test detach/attach with rbd cli
+DEV=`_sudo rbd device --device-type nbd --options try-netlink map ${POOL}/${IMAGE}`
+get_pid ${POOL}
+_sudo mount ${DEV} ${TEMPDIR}/mnt
+_sudo rbd device detach ${POOL}/${IMAGE} --device-type nbd
+expect_false get_pid ${POOL}
+expect_false _sudo rbd device attach --device ${DEV} ${POOL}/${IMAGE} --device-type nbd
+_sudo rbd device attach --device ${DEV} ${POOL}/${IMAGE} --device-type nbd --force
+get_pid ${POOL}
+_sudo rbd device detach ${DEV} --device-type nbd
+expect_false get_pid ${POOL}
+_sudo rbd device attach --device ${DEV} ${POOL}/${IMAGE} --device-type nbd --force
+get_pid ${POOL}
+ls ${TEMPDIR}/mnt/
+dd if=${TEMPDIR}/mnt/test of=/dev/null bs=1M count=1
+_sudo dd if=${DATA} of=${TEMPDIR}/mnt/test1 bs=1M count=1 oflag=direct
+_sudo umount ${TEMPDIR}/mnt
+unmap_device ${DEV} ${PID}
```

Comment:
If you look at what 4f6370b8843d3ddf2bcb0a53c7bf62037021014a did, it replaced `rbd-nbd` commands with `rbd device` commands.  `rbd-nbd attach` and `rbd-nbd detach` remained because there were no `rbd device attach` and `rbd device detach` equivalents.

So what I am looking for is not a new test case but just replacing `rbd-nbd attach` and `rbd-nbd detach` with the new commands in the existing test case and then adding those two lines for `rbd device attach` without `--force`.  Sorry for confusion!

Reply:
Sure no problem.